RSFontSizes/Classes/RSFontSizes.swift

```diff
@@ -179,7 +180,8 @@ extension Size {
     case .screen3_5Inch: return .screen4Inch
     case .screen4Inch: return .screen3_5Inch
     case .screen4_7Inch: return .screen5_5Inch
-    case .screen5_5Inch: return .screen4_7Inch
+    case .screen5_5Inch: return .screen4.7Inch
+    case .screen5_8Inch return .screen5_5Inch
```

Comment:
missing `:` after the case.

Reply:
Ok i will look it, i've done also some other edit because of the format of the iPhone x and it's strange size. i'll commit it soon

Reply:
Hi @kvacquier, can you please change the indentation to two spaces to match the project default? 
Also the tests failed because it couldn't find 1.0.1 version of the pod, I will do that once its updated. 
Besides that, everything cool :)